src/Features/Core/Portable/EditAndContinue/ManagedEditAndContinueCapabilities.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.EditAndContinue
+{
+    internal class ManagedEditAndContinueCapabilities
```

Comment:
>ManagedEditAndContinueCapabilities [](http://example.com/codeflow?start=19&length=34)

Is it necessary to wrap the enum into a class? Seems heavy weight.

Reply:
I"m not fussed about the class existing, but I do like the ease of `HasCapability(..)` over having to do a normal flags enum check everywhere.. unless we already have a helper for that? I guess alternatively that could just become an extension method.

Reply:
> I guess alternatively that could just become an extension method.

Yes. That'd be better. But I don't mind flag checks. There is also HasFlag on enum I think, but not sure how fast it is.

Reply:
Seems like it's an [intrinsic](https://source.dot.net/#System.Private.CoreLib/Enum.cs,337) in .NET Core, but not sure about .NET FX.

Reply:
According to a random blog post I found, it seems HasFlag is pretty quick even on .NET FX, so will just use that.